src/wyzeapy/types.py

```diff
@@ -102,7 +104,7 @@ class PropertyIDs(Enum):
     CONTACT_STATE = "P1301"
     MOTION_STATE = "P1302"
     CAMERA_SIREN = "P1049"
-    FLOOD_LIGHT = "P1056" # Also lamp socket on v3/v4 with lamp socket accessory
+    FLOOD_LIGHT = "P1056" # Also lamp socket on v3/v4 with lamp socket accessory. And also garage door state on v3 with garage door accessory.
```

Comment:
I wonder if we should rename this to CAMERA_ACCESSORY or something along those lines to avoid confusion since it can be multiple things now. If so we would need to make sure all references are updated.